apps/mobile/src/state/settings/settings.slice.ts

```diff
@@ -14,16 +14,20 @@ import { useAppDispatch } from '../utils';
 
 export type Theme = 'light' | 'dark' | 'system';
 
+export type WalletSecurityLevel = 'not-defined' | 'secure' | 'not-secure';
```

Comment:
```suggestion
export type WalletSecurityLevel = 'undefined' | 'secure' | 'insecure';
```

makes more sense I think?